utils/releaser/src/DependencyUpdater.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser;
+
+use Symplify\MonorepoBuilder\Composer\Section;
```

Comment:
in previous releases shopsys/ packages versions are without prefix `v` but releaser make it with `v` but composer works with both versions so this is just for consideration.

Reply:
We could trim the "v" in `DependencyUpdater`, however, I do not find this necessary as it works for both versions as you say :slightly_smiling_face: